repository/src/main/java/org/apache/atlas/tasks/TaskRegistry.java

```diff
@@ -98,8 +98,30 @@ public List<AtlasTask> getPendingTasks() {
             }
         } catch (Exception exception) {
             LOG.error("Error fetching pending tasks!", exception);
-        } finally {
-            graph.commit();
+        }
+
+        return ret;
+    }
+
+    @GraphTransaction
```

Comment:
I think no need to have @GraphTransaction here as this is the read operation
I am aware that this annotation is added at multiple other places where Read is performed, but I feel that we do not need to call GraphTransactionInterceptor, Let me know your thoughts

Reply:
since we are not writing graphTransaction is redundant.

Reply:
addressed